documentation/02_where.md

```diff
@@ -0,0 +1,19 @@
+---
+id: where
+title: Where to go? 
+sidebar_label: Where to go?
+---
+
+While a deep dive through the Leo documentation is a useful exercise, we realize that it may not be very pratical. To help you along your journey, we've provided a "map" to help you find your way.
+
+## Roadmap
+
+- If you're brand new to Leo, go to **Getting Started** and make your way through **Hello, Leo**.
+
+- If you're designing a dApp, we recommend checking out **Leo By Example** for a reference.
+
+- If you're interested in advanced concepts, check out **Concepts** and **Recipes**. Be sure to check out the **Core Aleo Concepts**.
```

Comment:
Concepts and Core Aleo Concepts are not a table of contents header. Perhaps one way to avoid failure is to make these links to the headers and then CI can identify issues?